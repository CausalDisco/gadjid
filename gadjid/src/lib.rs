// SPDX-License-Identifier: MPL-2.0
#![warn(missing_docs)]
//! gadjid -  Graph Adjustment Identification Distance library
mod ascending_list_utils;
mod graph_loading;
pub mod graph_operations;
mod partially_directed_acyclic_graph;

pub use graph_loading::constructor::EdgelistIterator;
pub use partially_directed_acyclic_graph::LoadError;
pub use partially_directed_acyclic_graph::PDAG;
