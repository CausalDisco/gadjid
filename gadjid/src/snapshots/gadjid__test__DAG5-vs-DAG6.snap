---
source: src/lib.rs
expression: "test(\"10005.DAG-100\", \"10006.DAG-100\")"
---
g_guess: 10006.DAG-100
g_true: 10005.DAG-100
ts:
  - 92
  - 53
  - 87
  - 77
  - 70
ys:
  - 38
  - 24
  - 56
  - 77
  - 9
zs:
  - - 2
    - 4
    - 5
    - 14
    - 25
    - 26
    - 28
    - 29
    - 30
    - 31
    - 34
    - 36
    - 44
    - 45
    - 53
    - 60
    - 69
    - 70
    - 74
    - 76
    - 87
  - - 5
    - 12
    - 24
    - 31
    - 42
    - 46
  - - 4
    - 9
    - 12
    - 21
    - 30
    - 38
    - 40
    - 41
    - 44
    - 54
    - 57
    - 64
    - 68
    - 69
  - - 1
    - 5
    - 7
    - 8
    - 12
    - 13
    - 16
    - 22
    - 27
    - 28
    - 31
    - 35
    - 38
    - 41
    - 48
    - 52
    - 63
    - 67
    - 69
    - 73
    - 76
  - - 1
    - 6
    - 7
    - 14
    - 17
    - 22
    - 28
    - 33
    - 34
    - 45
    - 46
    - 51
    - 59
    - 61
    - 63
    - 68
nams:
  - []
  - []
  - []
  - []
  - []
nvas:
  - - 2
    - 4
    - 5
    - 14
    - 25
    - 26
    - 28
    - 29
    - 30
    - 31
    - 34
    - 36
    - 44
    - 45
    - 53
    - 60
    - 69
    - 70
    - 74
    - 76
    - 87
  - - 5
    - 12
    - 24
    - 31
    - 42
    - 46
  - - 4
    - 9
    - 12
    - 21
    - 30
    - 38
    - 40
    - 41
    - 44
    - 54
    - 57
    - 64
    - 68
    - 69
  - - 1
    - 5
    - 7
    - 8
    - 12
    - 13
    - 16
    - 22
    - 27
    - 28
    - 31
    - 35
    - 38
    - 41
    - 48
    - 52
    - 63
    - 67
    - 69
    - 73
    - 76
  - - 1
    - 6
    - 7
    - 14
    - 17
    - 22
    - 28
    - 33
    - 34
    - 45
    - 46
    - 51
    - 59
    - 61
    - 63
    - 68
pa_true_1st_T:
  - 2
  - 4
  - 5
  - 14
  - 25
  - 26
  - 28
  - 29
  - 30
  - 31
  - 34
  - 44
  - 45
  - 53
  - 60
  - 69
  - 70
  - 74
  - 76
  - 87
an_true_1st_T:
  - 0
  - 1
  - 2
  - 3
  - 4
  - 5
  - 6
  - 7
  - 8
  - 9
  - 10
  - 11
  - 12
  - 13
  - 14
  - 15
  - 16
  - 17
  - 18
  - 19
  - 20
  - 21
  - 22
  - 23
  - 24
  - 25
  - 26
  - 27
  - 28
  - 29
  - 30
  - 31
  - 32
  - 33
  - 34
  - 35
  - 36
  - 37
  - 38
  - 39
  - 40
  - 41
  - 42
  - 43
  - 44
  - 45
  - 46
  - 47
  - 48
  - 49
  - 50
  - 51
  - 52
  - 53
  - 54
  - 55
  - 56
  - 57
  - 58
  - 59
  - 60
  - 61
  - 62
  - 63
  - 64
  - 65
  - 66
  - 67
  - 68
  - 69
  - 70
  - 71
  - 72
  - 73
  - 74
  - 76
  - 87
  - 92
ch_true_1st_T:
  - 97
de_true_1st_T:
  - 92
  - 97
poss_de_true_1st_T:
  - 92
  - 97
prop_an_true_1st_T_and_1st_Y:
  - 0
  - 1
  - 2
  - 3
  - 4
  - 5
  - 6
  - 7
  - 8
  - 9
  - 10
  - 11
  - 12
  - 13
  - 14
  - 15
  - 16
  - 17
  - 18
  - 19
  - 21
  - 22
  - 23
  - 24
  - 25
  - 27
  - 29
  - 30
  - 31
  - 35
  - 37
  - 38
shd:
  - 0.006262626262626263
  - 31
oset_aid:
  - 0.0017171717171717172
  - 17
parent_aid:
  - 0.01686868686868687
  - 167
ancestor_aid:
  - 0
  - 0
